examples/gnn_node.py

```diff
@@ -120,7 +132,8 @@ def train() -> float:
             task.entity_table,
         )
         pred = pred.view(-1) if pred.size(1) == 1 else pred
-        loss = loss_fn(pred, batch[entity_table].y)
+
+        loss = loss_fn(pred.float(), batch[entity_table].y.float())
```

Comment:
Do you mind changing `test_node_nn.py` accordingly to test your changes in this `gnn_node.py`?